astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
I'm a bit confused now, @cdeil @bmorris3 - As I see it the main use for this is: ``AirmassConstraint(2)``, meaning "airmass below 2, meaning the altitude should be *above* 30 degrees.  I think 99% of users will immediately recognize that an airmass constraint means "should be lower airmass/higher altitude than that".  But whether that is an `upper_limit` or `max` or `min`, I'm not sure because airmass and altitude have opposite senses...

But I just spent about 10 min talking with my observer officemates, and we couldn't come up with a pair of words that were unambiguous.  So just so long as ``AirmassConstraint(2)`` means "the object should have a secz value less than 1``, I'm happy ;)